src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components"
+
+export const Button = styled.button`
```

Comment:
There are also a few missing button styles:

- The letter spacing is `2.5px` (reuse your CSS variables).
- The `font-weight` is extra bold (`800`).